app/addons/activetasks/stores.js

```diff
@@ -28,6 +28,35 @@ define([
       this._pollingIntervalSeconds = 5;
       this.sortCollectionByColumnHeader(this._sortByHeader);
       this._backboneCollection = backboneCollection;
+      this.setIsLoading(true);
+    },
+
+    isLoading: function () {
+      return this._isLoading;
+    },
+
+    setIsLoading: function (bool) {
+      this._isLoading = bool;
+      if (bool) {
+        this.startTimerForLoading();
+      } else {
+        this.stopTimerForLoading();
+      }
+    },
+
+    startTimerForLoading: function () {
+      this._clockForLoadingLines = 0;
+      this._clockForLoadingLinesID = setInterval(function () {
+        this._clockForLoadingLines++;
```

Comment:
maybe this is silly, but: can we not just remember the time we started and compare it with the time of stop 2 determine whether or not we want to show the lines a little longer? (i.e. this._startTimeOfLoad=(new Date()).getTime() and if(((new Date()).getTime()-this._startTimeOfLoad)<x) then...